modules/mapper-extras/src/yamlRestTest/resources/rest-api-spec/test/rank_feature/10_basic.yml

```diff
@@ -29,6 +29,63 @@ setup:
           pagerank: 100
           url_length: 20
 
+  - do:
+      indices.create:
+        index: test1
+        body:
+          settings:
+            number_of_replicas: 0
+          mappings:
+            properties:
+              pagerank:
```

Comment:
Let's simplify this test: let's just have 2 fields: pagerank where `null_value` is set up, and `url_length` where it is not (impact only influences score calculation and is not relevant here)